paper-slider.html

```diff
@@ -90,6 +90,10 @@
       :host(:focus) {
         outline: none;
       }
+        
+      :host-context([dir="rtl"]) {
```

Comment:
`:dir(rtl)` would be great, but it's not yet supported

https://developer.mozilla.org/en-US/docs/Web/CSS/:dir#Browser_compatibility